src/pages/claim-details/claim-actions.tsx

```diff
@@ -82,56 +121,48 @@ export default function ClaimActions(props: Props) {
         </div>
       );
 
-    case 'Rejected':
-      if (claim.statusUpdatedBy === 'arbitrator') {
-        return (
-          <div className={styles.actionSection}>
-            <p>Kleros</p>
-            <div className={styles.actionMainInfo}>Rejected</div>
-            <div className={styles.actionPanel}>
-              <Button type="secondary" disabled={disableActions} onClick={handleAppeal}>
-                Appeal
-              </Button>
-            </div>
-          </div>
-        );
-      }
+    case 'DisputeResolvedWithClaimPayout':
+      return (
+        <div className={styles.actionSection}>
+          <p>Kleros</p>
+          <div className={styles.actionMainInfo}>Approved full amount</div>
+        </div>
+      );
 
+    case 'DisputeResolvedWithSettlementPayout':
       return (
         <div className={styles.actionSection}>
-          <p>API3 Multi-sig</p>
-          <div className={styles.actionMainInfo}>Rejected</div>
+          <p>Kleros</p>
+          <div className={styles.actionMainInfo}>
+            Approved <br />
+            counter of <br />
+            {formatApi3(claim.counterOfferAmount!)} API3
+          </div>
           <div className={styles.actionPanel}>
             <Button type="secondary" disabled={disableActions} onClick={handleAppeal}>
-              Escalate to Kleros
+              Appeal
             </Button>
           </div>
         </div>
       );
 
-    case 'Resolved':
-      if (claim.statusUpdatedBy === 'arbitrator') {
-        return (
-          <div className={styles.actionSection}>
-            <p>Kleros</p>
-            <div className={styles.actionMainInfo}>
-              Approved <br />
-              counter of <br />
-              {formatApi3(claim.resolvedAmount!)} API3
-            </div>
-            <div className={styles.actionPanel}>
-              <Button type="secondary" disabled={disableActions} onClick={handleAppeal}>
-                Appeal
-              </Button>
-            </div>
+    case 'DisputeResolvedWithoutPayout':
+      return (
+        <div className={styles.actionSection}>
+          <p>Kleros</p>
+          <div className={styles.actionMainInfo}>Rejected</div>
```

Comment:
Should this say `Rejected`? Shouldn't it be somethig like "resolved without payout"?